### DISCUSSION TITLE
Integrated with nuxtfirebase module (request)

### DISCUSSION THREAD

**Discussion by silvesterwali** [nuxt-community/auth-module#1609] [Ideas & Feature Requests]
i this module is really help . i hope this module is also can connected with firebase nuxt module https://firebase.nuxtjs.org/  as like other strategies ... so when use the createUserWithEmailAndPassword method or same auth method  from firebase nuxt module will work with auth module

**Comment by mklueh** [comment]
Thanks for opening this issue. I was also wondering if there is any way to combine both and allow Google / Facebook login

**Comment by Benjamin-Lee** [comment]
I would love to see this as well!
